src/components/custom/HeaderComponent.vue

```diff
@@ -1,7 +1,6 @@
 <template>
   <header
-    class="sticky top-0 z-50 w-full border-b bg-background/95 backdrop-blur supports-[backdrop-filter]:bg-background/60"
-  >
+    class="sticky top-0 z-50 w-full border-b bg-background/95 backdrop-blur supports-[backdrop-filter]:bg-background/60">
```

Comment:
Seems like the diff in this file is just from some different linting standards? Inside the `package.json`, I've been using the `npm run lint` & `npm run format` for linting, and we'll both want to use those so we can standardize the linting rules across contributors

Reply:
I will make sure to run both of these before making any commits or PRs 👍